src/pages/connections/farmlands.astro

```diff
@@ -0,0 +1,12 @@
+---
+import Farmlands from '../../components/Farmlands.vue';
+import BaseLayout from '../../layouts/BaseLayout.astro';
+
+const title = 'Farmlands';
+const description = 'Farmlands has partnered with Centrapay to deliver new ways for your business to authorise and process Farmlands Card payments that are faster, easier, and more secure than ever.';
+const img = 'countryside.jpg';
+---
```

Comment:
[Astro components](https://docs.astro.build/en/core-concepts/astro-components/) output pure html by default - this helps make our site snappy. This 'frontmatter' encapsulate Javascript that is only run at build time (it is not shipped to the client)